src/engraving/dom/edit.cpp

```diff
@@ -6244,6 +6244,11 @@ void Score::undoAddElement(EngravingItem* element, bool addToLinkedStaves, bool
                 }
                 Chord* c1 = findLinkedChord(cr1, score->staff(staffIdx));
                 Chord* c2 = findLinkedChord(cr2, score->staff(staffIdx + sm));
+
+                IF_ASSERT_FAILED(c1) {
```

Comment:
FWIW: 3.x does the same here, basically (it just returns when there's no c1)
Apparently a backport of #10654 which got closed in favor of #13183